apps/prairielearn/elements/pl-drawing/elements.py

```diff
@@ -36,28 +39,35 @@ def abserr_ang(ref, x):
 
 
 class BaseElement:
-    def generate(element, data):
+    @staticmethod
```

Comment:
Can you double check this doesn't cause any issues with element extensions? There should be some used by example course.

Reply:
![image](https://github.com/user-attachments/assets/3abdb782-5c06-464e-a1a1-4e88a22cfcce)